src/dump.js

```diff
@@ -60,7 +60,7 @@ export default ( function() {
 			parse: function( obj, objType, stack ) {
 				stack = stack || [];
 				var res, parser, parserType,
-					inStack = inArray( obj, stack );
+					inStack = stack.indexOf( obj );
```

Comment:
Nope, we need the actual index value later. Renamed this variable to make it clearer as to what is going on.